### DISCUSSION TITLE
workstation offline

### DISCUSSION THREAD

**Discussion by DevOpsPa1** [learndevopsonline/learndevopsonline#1836] [Q&A]
![Screenshot (33)](https://github.com/learndevopsonline/learndevopsonline/assets/157614154/d1fe4042-c547-4b88-9808-90657fe51d53)

**Comment by chaithanya1812** [comment]
can you share jenkins credntials?
and workstation-IP ?

Problem might be install java(11 or higher version) on workstation , it will work..

**Comment by DevOpsPa1** [comment]
Hi,
44.206.255.232 admin admin123
35.153.49.141 work station
 
But i have already installed java11

**Comment by chaithanya1812** [comment]
I missed it, I think you might be stopped the servers..

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/e751a7c2-dfce-4316-8ab8-c64bb8f12668)

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/e2b62147-9a76-44a5-87e7-b47919cbbc4c)


# No issues tomorrow we can solve it.

**Comment by DevOpsPa1** [comment]
Can you please tell your availability timings

Regards,
Pavan

**Comment by chaithanya1812** [comment]
You can join lab query session..

Or

### share the IP address(jenkins,workstation) once more time.. issue will be cleared

**Comment by DevOpsPa1** [comment]
Hi 
Thank you for your support  Chaithanya.
